### PR TITLE
update docs for ACP React Native plugins (1.x -> 2.x)

### PR DESCRIPTION
**Pull request by yangyansong-adbe**

### CODE REVIEW

File: getting-started/get-the-sdk.md [hunk 1]
```diff
@@ -89,11 +90,24 @@ You should see a pop-up similar to the following image:
 {% tab title="React Native" %}
 ### React Native
 
-Adobe Experience Platform Mobile SDK plugin for React Native supports React Native **versions 0.44.0 or later**. For the latest installation instructions, see the `README` file in the [`react-native-acpcore`](https://github.com/adobe/react-native-acpcore) repository.
+Adobe Experience Platform Mobile SDK plugin for React Native supports React Native **versions 0.60.0 or later**. For the latest installation instructions, see the `README` file in the [`react-native-acpcore`](https://github.com/adobe/react-native-acpcore) repository.
+
+{% hint style="info" %}
+
+The plugins start from v2.0.0 use [autolinking](https://github.com/react-native-community/cli/blob/master/docs/autolinking.md), which links plugins' native depdencies automatically. After installing the SDK plugins from npm, the extra step is needed on iOS:
```
Comment (cacheung) [inline getting-started/get-the-sdk.md:97]: Typo for "dependencies"

File: getting-started/get-the-sdk.md [hunk 2]
```diff
@@ -387,11 +401,83 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
+#import "ACPUserProfile.h"
+#import "ACPIdentity.h"
+#import "ACPLifecycle.h"
+#import "ACPSignal.h"
+...
+@implementation AppDelegate
+-(BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    [ACPCore setLogLevel:ACPMobileLogLevelDebug];
+    [ACPCore configureWithAppId:@"<your_environment_id_from_Launch>"];
+    [ACPUserProfile registerExtension];
```
Comment (cacheung) [inline getting-started/get-the-sdk.md:420]: Do we want to include userprofile in core example?

File: getting-started/get-the-sdk.md [hunk 3]
```diff
@@ -387,11 +401,83 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
+import com.adobe.marketing.mobile.InvalidInitException;
+import com.adobe.marketing.mobile.Lifecycle;
+import com.adobe.marketing.mobile.LoggingMode;
+import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.Signal;
+import com.adobe.marketing.mobile.UserProfile;
+...
+import android.app.Application;
+...
+public class MainApp extends Application {
```
Comment (cacheung) [inline getting-started/get-the-sdk.md:453]: Should it be
"public class MainApplication extends Application implements ReactApplication"
as what we have in the ACPCoreSampleApp?

File: getting-started/get-the-sdk.md (hunk 2 above)
Comment (yangyansong-adbe) [reply getting-started/get-the-sdk.md:420]: https://aep-sdks.gitbook.io/docs/getting-started/get-the-sdk#2-add-initialization-code
`
The following code snippets demonstrate how you can import and register the Mobile Core and Profile extensions. You can also see, for reference, how Identity, Lifecycle, Signal, Profile, and other extensions are imported and registered.
`
yes, we'd better keep it as we should also show the native code to register other extension.

Comment (cacheung) [reply getting-started/get-the-sdk.md:420]: ok

**Review by cacheung — APPROVED** [review]

**Review by shtomar-adb — APPROVED** [review]
LGTM!!

File: resources/upgrading-to-aep/current-sdk-versions.md [hunk 4]
```diff
@@ -114,7 +114,7 @@ Adobe Experience Platform Mobile SDK for Android supports Apple iOS 10 or later
 
 ## React Native
 
-Adobe Experience Platform Mobile SDK plugin for React Native supports React Native v**ersions 0.44.0 or later**. For the latest installation instructions, see the `README` file in the [`react-native-acpcore`](https://github.com/adobe/react-native-acpcore) repository.
+Adobe Experience Platform Mobile SDK plugin for React Native supports React Native v**ersions 0.60.0 or later**. For the latest installation instructions, see the `README` file in the [`react-native-acpcore`](https://github.com/adobe/react-native-acpcore) repository.
```
Comment (emdobrin) [inline resources/upgrading-to-aep/current-sdk-versions.md:117]: typo: `**version 0.60.0 or later**`

File: getting-started/get-the-sdk.md [hunk 5]
```diff
@@ -89,11 +90,24 @@ You should see a pop-up similar to the following image:
 {% tab title="React Native" %}
 ### React Native
 
-Adobe Experience Platform Mobile SDK plugin for React Native supports React Native **versions 0.44.0 or later**. For the latest installation instructions, see the `README` file in the [`react-native-acpcore`](https://github.com/adobe/react-native-acpcore) repository.
+Adobe Experience Platform Mobile SDK plugin for React Native supports React Native **versions 0.60.0 or later**. For the latest installation instructions, see the `README` file in the [`react-native-acpcore`](https://github.com/adobe/react-native-acpcore) repository.
```
Comment (emdobrin) [inline getting-started/get-the-sdk.md:93]: **version 0.60.0 or later**

File: release-notes/README.md [hunk 6]
```diff
@@ -4,6 +4,28 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## Aug 6, 2021
+
+### Adobe Experience Platform Mobile SDK plugins for React Native
+
+This major release introduces breaking changes that will bring:
```
Comment (emdobrin) [inline release-notes/README.md:11]: This major release introduces support for the following:

- Support for React Native version 0.60.+
- Autolinking for native dependencies and removal of the bundled SDK binaries (xcframework) from the React Native module.
- Dynamic versions for native dependencies to always load the latest SDK.
- Removal of several deprecated APIs.

Please note that this release introduces breaking changes. For more details, please see the release notes of the following plugins:

**Review by emdobrin — APPROVED** [review]

File: release-notes/README.md [hunk 7]
```diff
@@ -4,6 +4,28 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
...
+This major release introduces support for the following:
+
+- React Native version 0.60.+
+- Autolinking for native dependencies and removal of the bundled SDK binaries (xcframework) from the React Native module.
+- Dynamic versions for native dependencies to always load the latest SDK.
+- Removal of several deprecated APIs.
+
+Please note that this release introduces breaking changes. For more details, see the release notes of the following plugins:
+
+* [@adobe/react-native-acpcore  2.0.0](https://github.com/adobe/react-native-acpcore/releases/tag/v2.0.0)
```
Comment (calebk1) [inline release-notes/README.md:20]: Looks like there are some random extra spaces that slipped in here!

**Review by calebk1 — APPROVED** [review]
LGTM! Just one comment.

**Pull request merged**